lib/git_hooks/hook_installer.rb

```diff
@@ -0,0 +1,54 @@
+require 'erb'
+
+module GitHooks
+  class HookInstaller
+    HOOK_SAMPLE_FILE = 'hook.sample.erb'
+
+    def initialize(hook, ruby_path)
+      @hook = hook
+      @ruby_path = ruby_path
+    end
+
+    def install(force = false)
+      throw Exceptions::UnknownHookPresent.new(hook) if !force && installed?
+
+      hook_script = ERB.new(hook_template).result(binding)
+
+      puts "Writing to file #{hook_path}"
+      File
+        .open(hook_path, 'w')
+        .write(hook_script)
+
+      FileUtils.chmod(0775, hook_path)
```

Comment:
I think this is to make the hook an executable? @rranelli may be able to speak to this better